src/GPU_API/CUAPI_SendExtPotGREP2GPU.cu

```diff
@@ -0,0 +1,42 @@
+#include "CUAPI.h"
+#include "CUPOT.h"
+
+#if ( defined GPU  &&  defined GRAVITY )
+
+
+extern real   *h_ExtPotTable;
+extern double *h_GREP_Lv_Data_New;
+extern double *h_GREP_Lv_Radius_New;
+extern int     h_GREP_Lv_NBin_New;
+
+extern void CUAPI_SendExtPotTable2GPU( const real *h_Table );
+
+
+
+
+//-------------------------------------------------------------------------------------------------------
+// Function    :  CUAPI_SendExtPotGREP2GPU
+// Description :  Send the GREP potential table to GPU using the interface for EXT_POT_TABLE
+//
+// Note        :  1. Invoked by Poi_UserWorkBeforePoisson_GREP()
+//                2. EXT_POT_GREP_NAUX_MAX is defined in Macro.h (default = 4000)
+//                3. d_ExtPotTable[] and h_ExtPotTable[] are shared by both EXT_POT_TABLE and EXT_POT_GREP
+//-------------------------------------------------------------------------------------------------------
+void CUAPI_SendExtPotGREP2GPU()
+{
+
+   if ( h_GREP_Lv_NBin_New > EXT_POT_GREP_NAUX_MAX )
+      Aux_Error( ERROR_INFO, "Too many bins in GREP profiles %d !!\n", h_GREP_Lv_NBin_New );
+
+   for (int b=0; b<h_GREP_Lv_NBin_New; b++) {
+      h_ExtPotTable[b]                         = (real) h_GREP_Lv_Data_New  [b];
+      h_ExtPotTable[b + EXT_POT_GREP_NAUX_MAX] = (real) h_GREP_Lv_Radius_New[b];
+   }
```

Comment:
- Fill in `h_ExtPotTable[]` for the CPU-only case (i.e., when disabling `GPU` in the Makefile).